File: src/main/java/hu/frontrider/arcana/research/BiomancyScaneable.java
package hu.frontrider.arcana.research;

import net.minecraft.entity.player.EntityPlayer;
import thaumcraft.api.research.IScanThing;

public class BiomancyScaneable implements IScanThing {
    @Override
    public boolean checkThing(EntityPlayer entityPlayer, Object o) {
        return false;
    }

    @Override
    public String getResearchKey(EntityPlayer entityPlayer, Object o) {
        return null;
    }

    @Override
    public void onSuccess(EntityPlayer player, Object object) {

    }
}

File: src/main/java/hu/frontrider/arcana/research/BiomancyScaneable.kt
package hu.frontrider.arcana.research

import net.minecraft.entity.player.EntityPlayer
import thaumcraft.api.research.IScanThing

class BiomancyScaneable : IScanThing {
    override fun checkThing(entityPlayer: EntityPlayer, o: Any): Boolean {
        return false
    }

    override fun getResearchKey(entityPlayer: EntityPlayer, o: Any): String? {
        return null
    }

    override fun onSuccess(player: EntityPlayer?, `object`: Any?) {

    }
}
